src/cef/cef.go

```diff
@@ -4,21 +4,6 @@
 
 package cef
 
-/*
-CEF capi fixes
---------------
-1. In cef_string.h:
-    this => typedef cef_string_utf16_t cef_string_t;
-    to => #define cef_string_t cef_string_utf16_t
```

Comment:
No modification to the headers was required for compilation,
although I am not sure why it was required. My testing was
very shallow in the sense that I worried about being able to
start the browser and use it for my ends, I did not experiment with the API.